memstore/vector_party.go

```diff
@@ -412,8 +412,12 @@ func (vp *cVectorParty) SliceByValue(lowerBoundRow, upperBoundRow int, value uns
 // party.
 func (vp *cVectorParty) SliceIndex(lowerBoundRow, upperBoundRow int) (startIndex, endIndex int) {
 	if vp.GetMode() == common.HasCountVector {
-		startIndex = vp.counts.UpperBound(0, vp.length, unsafe.Pointer(&lowerBoundRow)) - 1
-		endIndex = vp.counts.LowerBound(startIndex, vp.length, unsafe.Pointer(&upperBoundRow))
+		startIndex = vp.counts.UpperBound(0, vp.length+1, unsafe.Pointer(&lowerBoundRow))
```

Comment:
why `vp.length+1` not `vp.length`..?

Reply:
because counts vector is one element longer